_includes/code/animation/basic_staggered_animation/main.dart

```diff
@@ -0,0 +1,204 @@
+// Copyright 2017 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that
+// can be found in the LICENSE file.
+
+import 'dart:async';
+
+import 'package:flutter/material.dart';
+import 'package:flutter/scheduler.dart' show timeDilation;
+
+class StaggerAnimation extends StatelessWidget {
+  StaggerAnimation({ Key key, this.controller }) :
+
+    // Each animation defined here transforms its value between the subset of
```

Comment:
This comment, which appears to be just as I wrote it, makes no sense.

I suppose this is better:
```
    // Each animation defined here transforms its value during the subset
    // of the controller's duration defined by the animation's interval.
    // For example the opacity animation transforms its value during
    // the first 10% of the controller's duration.
    //
    // Most of the Intervals cause the animations' transforms to be
    // "staggered", i.e. to start one after another.
```